tests/test_visitors/test_ast/test_complexity/test_function/test_raises.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.visitors.ast.complexity.function import (
+    FunctionComplexityVisitor,
+    TooManyRaisesViolation,
```

Comment:
It is better to import violation from violations, not visitors.

Reply:
The existing `test_returns.py`, `test_expressions.py` and `test_awaits_count.py` import their violations from the visitor. Shall we keep this as it is for consistency?

Reply:
You can also fix it there. We would one day run `mypy` on tests!